spec/client_spec.rb

```diff
@@ -84,8 +85,7 @@
       ad = client.ad("12345")
       expect(ad).to be_a Hashie::Mash
       expect(ad.count).to eq 2
-      expect(ad.ad_list[0].data.ad_id).to eq 12345
```

Comment:
This was failing because `client.ad` returns a single ad rather than an ad list. Could this have been failing before my changes?